### PR TITLE
I created a component Picture and i put this in dropdownWindows.

### PR DESCRIPTION
**Pull request by Sonya092**

### CODE REVIEW

File: src/components/Picture/index.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const [pictures, setPictures] = useState([]);
+  useEffect(() => {
+    axios.get(`http://localhost:5050/images`).then(({ data }) => {
+      setPictures(data);
+    });
+  }, []);
+  return (
+    <SPicture>
+      {pictures.map((picture) => {
+        return <img src={picture.url} alt="opthalmology" />;
```
Comment (KeerhemFR) [inline src/components/Picture/index.jsx]: The alt has to be {picture.description} so the alt match the image content

**Review by KeerhemFR — CHANGES_REQUESTED** [review]
Just need a change on the alt and will be good for me

File: src/components/Picture/index.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default function Pictures() {
   return (
     <SPicture>
       {pictures.map((picture) => {
-        return <img src={picture.url} alt="opthalmology" />;
+        return <img src={picture.description} alt="opthalmology" />;
```
Comment (KeerhemFR) [inline src/components/Picture/index.jsx]: You didn't read the message well : the src was good with {picture.url} . What needed to be change was the alt, with {picture.description}

**Review by KeerhemFR — CHANGES_REQUESTED** [review]

**Review by KeerhemFR — APPROVED** [review]

File: src/components/Picture/index.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import SPicture from './style';
+
+export default function Pictures() {
+  const [pictures, setPictures] = useState([]);
+  useEffect(() => {
+    axios.get(`http://localhost:5050/images`).then(({ data }) => {
```
Comment (Hacene010) [inline src/components/Picture/index.jsx:8]: Don't forget to hide your url intro a .env file

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**